description: fix a missing variable
author: Wengling Chen <feiyu2817@gmail.com>

--- a/src/chrome/browser/android/rlz/rlz_ping_handler.cc
+++ b/src/chrome/browser/android/rlz/rlz_ping_handler.cc
@@ -76,7 +76,7 @@ void RlzPingHandler::Ping(
   DCHECK_EQ(id.length(), 50u);
 
   GURL request_url(base::StringPrintf(
-      "https://%s%s?", rlz_lib::kFinancialServer, rlz_lib::kFinancialPingPath));
+      "https://%s%s?", "about:blank", rlz_lib::kFinancialPingPath));
   request_url = net::AppendQueryParameter(
       request_url, rlz_lib::kProductSignatureCgiVariable, "chrome");
   request_url =
