From: Wengling Chen <feiyu2817@gmail.com>
Date: Tue, 22 Dec 2020 21:48:23 +0200
Subject: tenporary fix for missing the override jar files

---
 third_party/android_deps/BUILD.gn |   15 ---------------
 1 file changed, 15 deletions(-)
 
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -325,12 +325,6 @@ android_aar_prebuilt("androidx_fragment_
     ":androidx_viewpager_viewpager_java",
   ]
   resource_overlay = true
-  deps += [ "//third_party/android_deps/local_modifications/androidx_fragment_fragment:androidx_fragment_fragment_prebuilt_java" ]
-
-  # Omit this file since we use our own copy, included above.
-  # We can remove this once we migrate to AndroidX master for all libraries.
-  jar_excluded_patterns = [ "androidx/fragment/app/DialogFragment*" ]
-
   ignore_proguard_configs = true
 }
 
@@ -519,15 +513,6 @@ android_aar_prebuilt("androidx_preferenc
     ":androidx_recyclerview_recyclerview_java",
   ]
   resource_overlay = true
-  deps += [ "//third_party/android_deps/local_modifications/androidx_preference_preference:androidx_preference_preference_prebuilt_java" ]
-
-  # Omit these files since we use our own copy from AndroidX master, included above.
-  # We can remove this once we migrate to AndroidX master for all libraries.
-  jar_excluded_patterns = [
-    "androidx/preference/PreferenceDialogFragmentCompat*",
-    "androidx/preference/PreferenceFragmentCompat*",
-  ]
-
   ignore_proguard_configs = true
 }
 
