diff --git a/components/gcm_driver/instance_id/android/java/src/org/chromium/components/gcm_driver/instance_id/InstanceIDBridge.java b/components/gcm_driver/instance_id/android/java/src/org/chromium/components/gcm_driver/instance_id/InstanceIDBridge.java
--- a/components/gcm_driver/instance_id/android/java/src/org/chromium/components/gcm_driver/instance_id/InstanceIDBridge.java
+++ b/components/gcm_driver/instance_id/android/java/src/org/chromium/components/gcm_driver/instance_id/InstanceIDBridge.java
@@ -124,7 +124,10 @@ public class InstanceIDBridge {
                             SubscriptionFlagManager.buildSubscriptionUniqueId(
                                     mSubtype, authorizedEntity),
                             flags);
-                    return mInstanceID.getToken(authorizedEntity, scope, extras);
+                    if (mInstanceID != null)
+                        return mInstanceID.getToken(authorizedEntity, scope, extras);
+                    else
+                        return "";
                 } catch (IOException ex) {
                     return "";
                 }
