From: Wengling Chen <feiyu2817@gmail.com>
Date: Sat, 22 Feb 2020 03:42:13 -0500
Subject: Disable contextual search by default

---
 chrome/browser/profiles/profile.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/chrome/browser/profiles/profile.cc
+++ b/chrome/browser/profiles/profile.cc
@@ -266,7 +266,7 @@ void Profile::RegisterProfilePrefs(user_
 #if defined(OS_ANDROID)
   registry->RegisterStringPref(
       prefs::kContextualSearchEnabled,
-      std::string(),
+      prefs::kContextualSearchDisabledValue,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
 #endif  // defined(OS_ANDROID)
   registry->RegisterBooleanPref(prefs::kSessionExitedCleanly, true);
